99_ignore/new/ML8000_Become_an_Expert.md

```
@@ -0,0 +1,33 @@
+# Machine Learning / Deep Learning Forløb
+
+Hvis en af de følgende links ikke længere er gyldigt venligst sig det til læreren.
+
+## Bliv en ekspert i maskinlæring på højeste niveau
+
+## How I’d learn ML (if I could start over)
+
+https://www.youtube.com/watch?v=_xIwjmCH6D4
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)